### DISCUSSION TITLE
Meeting recording and publication

### DISCUSSION THREAD

**Discussion by csarven** [w3c-cg/solid#51] [Ideas]
Follows action of https://github.com/solid/specification/blob/main/meetings/2024-01-09.md#minutes-for-cg-going-forward

Background:
* https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md
* https://github.com/w3c-cg/solid/blob/main/meetings/README.md
* https://github.com/w3c-cg/solid/blob/main/meetings/template.md

Currently:
 * a public instance of [Jitsi](https://meet.jit.si/) (preferably one in which no account creation is required) for the meeting session. For example, the CG uses https://meet.jit.si/solid-cg .
* conversation notes (sometimes referred to as minutes) are taken during meetings using a collaborative editor such as [Hackmd](https://hackmd.io/) (preferably only signed-in users can edit) during the meeting.
* the creation of the agenda, minuting, reviewing, and publishing of minutes follow a process along the lines of https://github.com/w3c-cg/solid/pull/17
* minutes are published and available at GitHub for example in the https://github.com/solid/specification repository. There is no particular persistence/backup guarantee or plan, in addition to `github.com` essentially being a "third-party" space.

Going forward, the Group to consider using W3C infrastructure to help transcribing and publishing meeting minutes. (Audio/video can still be carried out using Jitsi for the foreseeable future).

W3C provides an IRC server for chats and meetings. More details at https://www.w3.org/Project/IRC/

The W3C IRC channel for the Solid CG is registered [#solid](irc://irc.w3.org/solid), and tooling such as [RSSAgent](https://www.w3.org/2002/03/RRSAgent) to help recording sessions and [Zakim](https://www.w3.org/2001/12/zakim-irc-bot.html) are available, and have been widely used in the W3C community for decades.

Transitioning to use the IRC will:
* help take advantage of automating common actions related to the meeting and publishing of minutes under `w3.org` space which has some assurance for persistence.
* help CG participants to be more familiar with the existing W3C infrastructure, and can take advantage of this in some other W3C Groups, including WGs.

**Comment by TallTed** [comment]
> * the meetings are transcribed using a collaborative editor such as [Hackmd](https://hackmd.io/) (preferably only signed-in users can edit) during the meeting.

Transcription — that is, word-for-word speech-to-text — whether automated or human-powered — hasn't worked well in any WG, CG, or other, in which I've been involved.

Rather, 
* conversation notes (sometimes referred to as minutes) are taken during meetings using a collaborative editor such as [Hackmd](https://hackmd.io/) (preferably only signed-in users can edit) during the meeting.

**Comment by csarven** [comment]
Good catch. Updated my comment with your suggestion FWIW.

**Comment by elf-pavlik** [comment]
I haven't seen much interest in CG using IRC, if we can't come up with an action that would resolve this issue I would propose to convert it into discussion. Once we have something actionable we can convert it back into issue or create actionable issue linking to the discussion.

**Comment by csarven** [comment]
According to the group's minutes, the last time this topic was discussed was on [2024-07-03](https://github.com/solid/specification/blob/main/meetings/2024-07-03.md#meeting-recording-and-publication).

The general sense of the room on this topic (in minutes and in this issue) was that there was no objection to trying out IRC, and some participants supported giving it a try. That should have been a clear action item. If it was not, and needed further discussion, why wasn't this issue added back to the agenda since 2024-07-03? Chairs have some responsibility to ensure participants are supported in this process.

Was there a single meeting held over IRC? Any related documentation, discussion, or training? If not, why not?

That seems like the point where this thread should continue, rather than converting it into a discussion, which does not change anything. Since none of the current or previous chairs seems to have followed up or took steps to use W3C IRC for the CG, is the expectation that moving this "issue" to a "discussion" will actually bring it more attention in meetings or lead to progress? Reminder that [this topic has already been raised six times in meetings](https://github.com/search?q=repo%3Asolid%2Fspecification%20https%3A%2F%2Fgithub.com%2Fw3c-cg%2Fsolid%2Fissues%2F18&type=code) and possibly in other contexts.

This is not a matter of needing more discussion, but of following up on previous ones by actually using W3C IRC in the CG.

---

That said, if the chairs wish to set aside prior discussions and views and revisit this topic given the different composition of chairs and participants, that is their decision. The proposal, however, still stands: make greater use of W3C infrastructure, and ensure the group's position is publicly recorded. If there is an objection, this issue can be <ins>closed.</ins>

**Comment by csarven** [comment]
I closed the issue by mistake as I was trying to finish my sentence above with the word "closed" and mistakenly clicked on GitHub's "close" button (when I actually wanted to see the options). In any case, feel free to reopen (since I don't have access) or do whatever you want.

**Comment by elf-pavlik** [comment]
> This is not a matter of needing more discussion, but of following up on previous ones by actually using W3C IRC in the CG.

@csarven if you plan to join one of upcoming CG meetings, take role of a scribe and use IRC, I'm happy to do it during one of the meetings I will chair and I guess none other chair would object to giving it a try during one of the meetings their chair. Even if  you would like to do it on a short notice, for example tomorrow I think we could make it happen.

**Comment by csarven** [comment]
I'll happily contribute as a scribe (as I have a lot in the past: https://virginiabalseiro.github.io/solid-ecosystem-monitor/ ) when I'm present, but I think I'm not a dependency to carry this particular issue out. Again, chairs should be leading this.

**Comment by elf-pavlik** [comment]
>  Again, chairs should be leading this.

I can't think of anyone other than you who would like to use IRC. If you don't want to lead that experiment, maybe you are aware of someone else who would like to use IRC and lead it?

I think charis should help remove obstacles that stop participants from making their ideas happen, not necessarily execute all the ideas that CG participants come up with.

Unless we can specify an action that resolves this issue and find someone who volunteers to assign themselves. I think *ideas* section of the discussions can accomodate this topic better https://github.com/w3c-cg/solid/discussions/categories/ideas

**Comment by csarven** [comment]
The point is about making use of W3C infrastructure *because* the current CG chairs were evidently not doing the bare minimum or correctly. So, the point was to simplify the work even further by taking advantage of the W3C infrastructure because it provides the tooling to help publish minutes!

Aside: why run for chair when reluctant to do work that requires following charters, processes, guidelines?

And, 1) if you read prior discussions and how this issue came to be, I am not the only one, and 2) there were no objections.

**Comment by elf-pavlik** [comment]
Can we stay focused on resolving this specific issue?
I propose following, due to this issue currently not being actionable:
* Convert it into discussion in *ideas* category
* Once it becomes actionable and there are clear acceptance criteria for resolving it, we convert it back into issue.

My main problem is that this repo's issue tracker may become similar to other repos in @solid org, for example https://github.com/solid/specification/issues has almost 200 open issues, and in 2025 at most a handfull were resolved.

> [csarven](https://github.com/csarven)added enhancement question [on Feb 17, 2024](https://github.com/w3c-cg/solid/issues/18#event-11836070730)

I think those labels fit better into discussions, and once something actionable emerges it can be tracked as issue that someone should be working on resolving. I should probably start a new discussion focused on how do we want to use issue tracker and discussions in this repo. Since now I'm diverging from the topic.

**Comment by csarven** [comment]
It looks like we are back to this https://github.com/w3c-cg/solid/issues/18#issuecomment-3517626631 .
